src/NServiceBus.Persistence.DynamoDB/Saga/SagaPersister.cs

```diff
@@ -13,6 +13,9 @@
 
     class SagaPersister : ISagaPersister
     {
+        const string SagaDataVersionAttributeName = "___VERSION___";
+        const string SagaLockAttributeName = "NSB_lease_timeout";
```

Comment:
for consistency (to be addressed later anyway), let's make this consistent with version and use something like `___LEASE_TIMEOUT____`